python/ray/autoscaler/sdk.py

```diff
@@ -102,7 +108,8 @@ def rsync(cluster_config: Union[dict, str],
           down: bool,
           ip_address: str = None,
           use_internal_ip: bool = False,
-          no_config_cache: bool = False):
+          no_config_cache: bool = False,
+          all_nodes: bool = False):
```

Comment:
This isn't something I want to expose in the SDK, since it inherently has a race condition with scaling up. Instead, you can use file mounts to sync files consistently to all nodes. What is the use case you need it for?

Reply:
Hmm, that is a fair point, but that means the CLI also has this problem--right? 

Would it make more sense to modify `rsync` to check that nodes are `up` before rsync'ing to them?

Reply:
It's exposed there, but only as a best effort kind of thing for clusters that are up. I want to discourage users from using that option however (maybe we should even remove it).

It's not a reliable building block given that clusters are inherently autoscaling.

Reply:
@ericl , removed!